packages/babel-plugin/src/styled/__tests__/tagged-template-expression.test.tsx

```diff
@@ -661,14 +661,20 @@ describe('styled tagged template expression', () => {
       import { styled } from '@compiled/react';
 
       const ListItem = styled.div\`
```

Comment:
The expected output is
```
expect(actual).not.toInclude('propz.loading?colors.N100:colors.N200');

 expect(actual).toIncludeMultiple([
    'isLoading ? colors.N20 : colors.N40',
    'l ? colors.N50 : colors.N10',
    'props.loading ? colors.N100 : colors.N200',
  ]);
```

The previous example is not longer relevant; the fix will generate classes instead of inline style to evaluate at run time.

The fix is not applied for certain cases of nested conditional rules; hence, I slightly changed the example to preserve the expected output.